api/models.go

```diff
@@ -430,7 +430,8 @@ type (
 
 	Source struct {
 		models.Source
-		I18n map[string]*models.SourceI18n `json:"i18n"`
+		I18n       map[string]*models.SourceI18n `json:"i18n"`
+		SourceCUID int64                         `json:"cuid,omitempty"`
```

Comment:
Source should not be changed. It's an atomic building block

Reply:
fixed